### ISSUE TITLE
Feature request: an option to print a hairline separating logical pages

### ISSUE THREAD

**Issue by alexeymuranov** [loopspace/pgfmorepages#6]
I am using this package and I am missing an option to print a line separating logical pages in n-up layouts. That would be helpful when I need to manually cut a page into logical pages.

**Comment by loopspace** [comment]
When a page is constructed then the package puts a rectangular path around it and you can specify `border code` to take advantage of that which might do what you want.  There are examples of this in the `examples` directory, such as in https://github.com/loopspace/pgfmorepages/blob/master/examples/repeated_layouts.tex

Does that help?

**Comment by alexeymuranov** [comment]
@loopspace , indeed, `border code=\pgfusepath{draw}` is good enough for me, thanks. I searched the documentation for "line," but not for "border." IMO it needs to be better documented. I'd like to know how to make the lines thinner, for example.

**Comment by Oreolek** [comment]
Why doesn't it work after the first page? Only the first page has this border.

**Comment by loopspace** [comment]
@Oreolek Could you post some code that shows that not happening?  My experiments aren't showing a problem so it's hard to diagnose what the problem might be.
